ui/job-view/headerbars/TierIndicator.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonGroup, Button } from 'reactstrap';
+
+const TIERS = ['1', '2', '3'];
+
+export default function TierIndicator(props) {
+  const { filterModel } = props;
+  const shownTiers = filterModel.urlParams.tier || [];
+
+  return (
+    <ButtonGroup>
+      {TIERS.map(tier => {
+        const isOnlyTier = shownTiers.length === 1 && tier === shownTiers[0];
```

Comment:
You could create a new const (perhaps called `isOnlyText`) up here that is either blank or has the "Must have at least..." text in it.  Then in the `title` param, you would just include the value of this new const, which might be blank.  But also have the `title` value I mentioned below.